src/ensembl/src/content/app/entity-viewer/shared/helpers/entity-helpers.ts

```diff
@@ -1,8 +1,20 @@
-import { SliceWithLocationOnly } from 'src/content/app/entity-viewer/types/slice';
+import {
+  Slice,
+  SliceWithLocationOnly
+} from 'src/content/app/entity-viewer/types/slice';
 
 export const getFeatureCoordinates = (feature: {
   slice: SliceWithLocationOnly;
 }) => {
   const { start, end } = feature.slice.location;
   return { start, end };
 };
+
+export const getFeatureStrand = (feature: { slice: Slice }) =>
+  feature.slice.region.strand;
```

Comment:
Perhaps return feature.slice.region.strand.code? There's nothing else we will be interested in regarding strand.

The consumer of the function can call the return value ("forward"/"reverse") "strand" (as in `const strand = getFeatureStrand(feature);`), to simplify the code slightly.

Reply:
✔️